src/core/public/keyboard_shortcut/keyboard_shortcut_service.ts

```diff
@@ -129,9 +121,16 @@ export class KeyboardShortcutService {
   private shouldIgnoreKeyboardEventForTarget(target: EventTarget | null): boolean {
     if (!this.isHTMLElement(target)) return false;
 
-    const tagName = target.tagName;
+    const element = target;
+    const tagName = element.tagName;
+
+    const inputTags = ['INPUT', 'TEXTAREA', 'SELECT'];
```

Comment:
Do we also need to check for contenteditable elements? I think we don't have contenteditable components. Will we put some concerns for future expansion? or we are focusing on current osd?